pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -861,7 +861,7 @@ let
           name = "phptools-vscode";
           publisher = "devsense";
           version = "1.33.13032";
-        } // sources.${stdenv.system};
+        } // sources.${stdenv.system} or throw "Unsupported system: ${stdenv.system}";
```

```suggestion
        } // sources.${stdenv.system} or (throw "Unsupported system: ${stdenv.system}");
```

Everyone will make this mistake given enough exposure to `or`.
 - https://github.com/NixOS/nix/issues/7405